supporting_content/dimensions/tags.md

```diff
@@ -0,0 +1,44 @@
+# Column: Tags
+
+## Example provider mappings
+
+Current resource types found or extracted from available data sets:
+
+| Provider  | Dataset                 | Column                                      | Hierarchical Resources | Supports Inheritance?
+| :-------- | :---------------------- | :-------------------------------------------| :-------------------------------------------| :----------
+| AWS       | CUR                     | resourceTags/user:\*, costCategories/\*       | Organization, Organizational Unit(s), Account | No
+| GCP       | BigQuery Billing Export | tags, labels, system_labels, project.labels 									 	|  Folder(s), Project | Yes
+| Microsoft | Cost details            | Tags | Subscription, Resource Group | Yes
```

Comment:
Not sure if the hierarchy is about the cloud or the billing data. If it's what's in billing data, then management group should not be in this list. Including it for completeness just in case.
```suggestion
| Microsoft | Cost details            | Tags | Billing Account, Billing Profile, Invoice Section, Department, Enrollment Account, Management Group, Subscription, Resource Group | Yes
```